src/components/GlobalStyle.tsx

```diff
@@ -0,0 +1,14 @@
+import { createGlobalStyle } from 'styled-components'
```

Comment:
**Nitpick**

Not sure this file needs to have the `.tsx` extension since it doesn't contain any JSX. Not sure it hurts either